src/packages/@ncigdc/components/UserDropdown.js

```diff
@@ -36,16 +36,23 @@ const DropdownItemStyled = styled(DropdownItem, {
 });
 
 const logout = () => {
+  const isDev = process.env.NODE_ENV === 'development';
```

Comment:
can you set this as an exported constant in constants.js like https://github.com/NCI-GDC/portal-ui/blob/85a91556f92d603072fa5b16f4e71f2173d914c0/src/packages/%40ncigdc/utils/constants.js#L21 then you don't need to keep redefining it from process.env